apps/projects/app/components/Content/IssueDetail/Label.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { textStyle, unselectable, useTheme } from '@aragon/ui'
+
+const Label = ({ text, block, children }) => {
+  const theme = useTheme()
+  return (
+    <label
+      css={`
+        ${textStyle('label2')};
+        ${unselectable()};
+        color: ${theme.surfaceContentSecondary};
+        display: ${block ? 'block' : 'initial'};
+      `}
+    >
+      {text}
+      {children && children}
+    </label>
+  )
+}
+
+Label.defaultProps = {
+  block: false,
```

Comment:
If block is required we shouldn't need a defaultProp, anyway consider using a more conventional and flexible "display" prop instead

Reply:
As above.